src/datasources/targeted-messaging/targeted-messaging.datasource.ts

```diff
@@ -108,6 +140,12 @@ export class TargetedMessagingDatasource
       );
       throw new UnprocessableEntityException('Error creating submission');
     });
+    const { key } = CacheRouter.getSubmissionCacheDir({
+      outreachId: args.targetedSafe.outreachId,
+      safeAddress: args.targetedSafe.address,
+      signerAddress: args.signerAddress,
+    });
```

Comment:
Shall we also make a `getSubmissionCacheKey` like we did for targeted Safes?

Reply:
Done in [ce9f899](https://github.com/safe-global/safe-client-gateway/pull/2000/commits/ce9f89920db5be3645b0c7dd3d38c847e2a34800)